packages/mobx-react/__tests__/finalizationRegistry.tsx

```diff
@@ -0,0 +1,88 @@
+import { cleanup, render, waitFor } from "@testing-library/react"
+import * as mobx from "mobx"
+import * as React from "react"
+
+// @ts-ignore
+import gc from "expose-gc/function"
+import { observer } from "../src"
+
+afterEach(cleanup)
+
+function sleep(time: number) {
+    return new Promise<void>(res => {
+        setTimeout(res, time)
+    })
+}
+
+// TODO dunno why it's not available
```

Comment:
project was missing a configuration file, please take a look https://github.com/mobxjs/mobx/pull/3620
or simply add the file manually.